### PR TITLE
Pay payment request returns payment activity

### PR DESCRIPTION
**Pull request by alexBinns**
Paying a payment request now returns a Payment Activity instead of a Payment Request to maintain consistency with refunding a Payment Request which returns a Payment Activity. Returning a Payment Activity also allows the poster to know what has changed. 
The model that describes the Payment Activity is on another page to the pay payment request route this moves the description for pay and refund payment request to the Payment Activity page.

### CODE REVIEW

**Comment by ncjones** [comment]
Moving the pay endpoint docs to the payment activities page is not appropriate.

File: src/api/payment-requests/payment-requests.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ version (documented on this page) and the "legacy" version (documented at
 
 ★  For payment options which specify an address, there's a requirement to make a transaction on an
 external ledger. Once you have made that payment, you can use the transaction id for
-[Paying a Payment Request][].
+[Paying a Legacy Payment Request][].
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:101]: I think this needs rewording. Legacy belongs in here, but maybe not in the link.

File: src/api/payment-requests/payment-requests.md [hunk 2]
```diff
@@ -478,139 +478,11 @@ them to find the Payment Request and proceed to pay.
...
-| 403    | {% break _ INVALID_MERCHANT_CONFIG %}  | The merchant is not configured properly to satisfy the payment request. This could be due to incorrect information, or the merchant’s credentials might be blocked by an external service. |
-| 403    | {% break _ QUOTA_EXCEEDED %}           | The payment pay request exceeds the allowed spend quota supplied.                                                                                                                          |
-| 403    | {% break _ INSUFFICIENT_ASSET_VALUE %} | The asset has insufficient funds to pay the payment request or the transaction amount received by Centrapay is less than the total of the payment.                                         |
-| 403    | {% break _ ASSET_REDEMPTION_DENIED %}  | The asset redemption has been unsuccessful due to an error with provided payment parameters, the merchant, or the asset.                                                                   |
+To pay a payment request refer to the [Pay Payment Request][] section on the [Payment Activities][] page
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:481]:
```suggestion
You can pay by creating a Payment Activity against the Payment Request. Please refer to the [Pay Payment Request][] section on the [Payment Activities][] page.
```

File: src/api/payment-requests/payment-requests.md [hunk 3]
```diff
@@ -478,139 +478,11 @@ them to find the Payment Request and proceed to pay.
...
-| 403    | {% break _ INACTIVE_ASSET %}                | The asset is not refundable. It may have been disabled, expired, or already refunded.                                                                                                                                                                                       |
-| 403    | {% break _ REFUND_NOT_SUPPORTED %}          | The asset type does not support refunds.                                                                                                                                                                                                                                    |
-| 403    | {% break _ REFUND_WINDOW_EXCEEDED %}        | The time since the payment exceeds the window of time a payment request can be refunded in.                                                                                                                                                                                 |
+To refund a payment request refer to the [Refund Payment Request][] section on the [Payment Activities][] page.
```
Comment (heymatthew) [inline src/api/payment-requests/payment-requests.md:485]:
```suggestion
You can refund by creating a Payment Activity against the Payment Request. Please refer to the [Refund Payment Request][] section on the [Payment Activities][] page.
```

**Review by heymatthew — APPROVED** [review]
Do these things and you're good with me.

**Comment by heymatthew** [comment]
>  Move the endpoint back to the main payment request page.

We decided this together and this is a reversal on what we agreed on. Happy to discuss, @ncjones please come get me if you disagree with the documentation changes.

**Comment by heymatthew** [comment]
I've had a talk to Nate. To merge this, you’ll need to

1. Inline the whole Payment Activities section
2. Move model descriptions to the top of the Payment Requests page
3. Setup a redirect from the old Payment Activities page to the anchor on the Payment Requests page

After that, ping him to remove his block and you'll be able to merge this :shipit:

**Review by ncjones — APPROVED** [review]

**Pull request merged**